src/app.js

```diff
@@ -94,4 +94,4 @@ app.use(compress())
 	.configure(middleware)
 	.configure(allHooks);
 
-module.exports = app;
+module.exports = app;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4791331)